src/DependencyInjection/Compiler/ResolveTargetEntityPass.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the user bundle package.
+ * (c) Connect Holland.
+ */
+
+namespace ConnectHolland\UserBundle\DependencyInjection\Compiler;
+
+use ConnectHolland\UserBundle\Entity\UserInterface;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+
+/**
+ * @codeCoverageIgnore WIP
```

Comment:
Should have an explanation of why it's ignored, or it should be removed and the tests should cover it.